### ISSUE TITLE
Contract managment with users and organization teams.

### ISSUE THREAD

**Issue by jondo89** [Raltrwx/Mercetplace#1]
Try our other post , which is the follow up on the fratenate page, it is for the engineering enviorment, however the long term intention is a contract builder , difficult to explian , but i suspect may be what your are looking to achieve.
https://heavy-lifting.herokuapp.com/

**Comment by jondo89** [comment]
p.s it is definitly a work in progress.

**Comment by MatthewABrantley** [comment]
Yes, it very well may be, could save me a lot of time on defining more of the routing logic, table display, and mongo models which has been the most challenging part for me. Thank you very much! I will look into re-building directly from Heavy Lifting most likely. 

My recaptcha always thinks I'm a robot as well, would you have any links with info on that? I've not found anything, I already added localhost to my allowed URL.

**Comment by jondo89** [comment]
![capture](https://user-images.githubusercontent.com/2895399/28963714-6879881a-790a-11e7-9484-730b79e2ab5b.PNG)
the domain needs to read "localhost"
the .env file needs to be changed to have your secret etc included.
Then likley the issue , the page with the recaptch div needs to have the correct sitekey!

`<div class="g-recaptcha" data-callback="enableBtn" data-sitekey="6Le2aCcUAAAAAO8g693E9uMACIv7L-DeAbDZc67-"></div>`

Just a note on the heavy-lifting site , it is really a alpha version , and you will likley get stuck, I am planning on completing the team managment first , then rebuilding it from scratch with the integrated team managment and commercialization .

**Comment by MatthewABrantley** [comment]
That's okay, I'm fine with experimenting, this is half a learning exercise for me. I can tell it's more alpha, but I may pull what I want from both projects. Thank you for the excellent boilerplate!

The recaptcha "works", I get the images to click, it just then always kicks me to /signup and it says I'm a robot. I'll keep googling.

**Comment by MatthewABrantley** [comment]
https://github.com/Raltrwx/mercetplace2 

If you're curious, just quickly smashed them together. Thanks again.
